### PR TITLE
Remove unused imports in Java files

### PR DESCRIPTION
**Pull request by leighmcculloch**
### What
Remove unused imports in Java files when saving the file.

### Why
To keep the list of imports relevant to the code in the file, and to
align with our IntelliJ config.

### Checklist

- [ ] ~Send RFC email to Braintree developers _if change may be impactful_. Please include a link to this PR so that discussion about the pros and cons of the change remains linked to the proposed changes. **Recent examples include**: addition of linter and completer, no longer removing end-of-line whitespace on save, changing to fzf for file lookup.~

### CODE REVIEW

File: vimrc.bundles [hunk 1]
```diff
@@ -75,6 +75,7 @@ Plug 'vim-scripts/groovyindent-unix'
 Plug 'vim-scripts/mako.vim'
 Plug 'vim-scripts/matchit.zip'
 Plug 'voxpupuli/vim-puppet', { 'commit': 'e88c19bf10763b30f86b7417677f59a9c9487fa2' }
+Plug 'akhaku/vim-java-unused-imports'
```
Comment (pgr0ss) [inline vimrc.bundles]: I think this list was alphabetized?

Comment (leighmcculloch) [reply vimrc.bundles]: be06292

**Comment by leighmcculloch** [comment]
The plugin doesn't handle wildcard imports well, which we have in our Java SDK. I've got an open PR on the plugin to fix that. akhaku/vim-java-unused-imports#20 Will hold merging this PR until that's resolved.

**Comment by leighmcculloch** [comment]
Fix got merged and it looks like it works great with a couple of our projects so I'm merging this now!

**Pull request merged**